src/main/scala/com/highperformancespark/robinsparkles/ComputePartitions.scala

```diff
@@ -53,11 +68,11 @@ case class ComputePartitions(val sparkConf: SparkConf) {
    * @param previousRuns
    * @return
    */
-  def fromStageMetric(previousRuns: List[WebUIInput]): Int = {
+  def fromStageMetric(previousRuns: List[StageInfo]): Int = {
     val concurrentTasks = possibleConcurrentTasks()
     previousRuns match {
       case Nil => concurrentTasks
-      case first :: Nil => (first.numPartitionsUsed + first.numPartitionsUsed)
+      case first :: Nil => (first.numPartitionsUsed + first.numExectutors)
```

Comment:
Does this take the sum of numPartitionsUsed and numExecutors?
Aside from the minor misspelling,
I'm not clear on why we would take this sum.

Reply:
aha, that makes sense
do you want to define the lower and upper bounds for number of partitions?
(initialize: 2/3 the number of cores, 
upper bound: 9/10 the number of cores
lower bound: ?)